spaces/domains.json

```
@@ -403,4 +403,5 @@
   "governance.yfdai.finance": "yfdaifinance.eth",
   "vote.samot.club": "samotclub.eth",
   "dictatorship.levxdao.org": "levx.eth"
+  "dao.dehero.io": "deherodao.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dictatorship.levxdao.org": "levx.eth",
  "dao.dehero.io": "deherodao.eth"
```